include/amiga/clib/cybergraphics_protos.h

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
system specific headers has no place in the project source tree. Vampire Lib should be included with the gcc6 toolchain, not in all the projects that uses it.